amlip_docs/rst/user_manual/nodes/async_edge.rst

```diff
@@ -0,0 +1,58 @@
+.. include:: ../../exports/alias.include
+
+.. _user_manual_nodes_async_edge:
+
+######################
+Asynchronous Edge Node
+######################
+
+This node is able to send data serialized as :ref:`user_manual_datatype_inference` and it receives an Inference as :ref:`user_manual_datatype_inference_solution`.
+
+Example of Usage
+================
+
+Users can use method :code:`request_inference` to send new data.
+The thread calling this method will wait until the whole process has finished and the *Inference* has arrived from the :ref:`user_manual_nodes_async_inference` in charge of this data.
+By destroying the node every internal entity is correctly destroyed.
+
+Steps
+-----
+
+* Instantiate the Edge Node creating an object of such class with a name, a listener and a domain.
+* Create a new :code:`InferenceDataType` from an array of bytes.
+* Send a data synchronously calling :code:`request_inference`.
+* Wait for the inference.
+
+.. tabs::
+
+    .. tab:: Python
+
+        .. code-block:: python
+
+            # Variable to wait to the inference
+            waiter = BooleanWaitHandler(True, False)
```

Comment:
Do not add WaitHandler to this test. Instead just comment that user must do something here what the solution.